Project.toml

```
@@ -35,11 +35,11 @@ MiniQhull = "0.1"
 Optim = "0.15, 0.16, 0.17, 0.18, 0.19, 0.20"
 Plots = "0.25, 0.26, 0.27, 0.28, 0.29"
 Polyhedra = "0.5"
-RecipesBase = "0.6, 0.7, 0.8"
+RecipesBase = "0.6, 0.7, 0.8, 1.0"
 Reexport = "0.2"
 Requires = "0.5, 1.0"
 TaylorModels = "0.3"
-julia = "1.0, 1.1, 1.2, 1.3"
+julia = "1.0, 1.1, 1.2, 1.3, 1.4"
```

Note that this is equivalent to
```suggestion
julia = "1.0"
```

---

As soon as v1.5 arrives, it is not equivalent anymore. This is a philosophical question (do you want to support non-existing Julia versions at the risk of not working then or do you want to play safe at the cost of having to make a new release for every Julia version).

---

> As soon as v1.5 arrives, it is not equivalent anymore. 

Well, I can install LazySets v1.34.0 in Julia 1.4.1

> do you want to support non-existing Julia versions at the risk of not working then

Minor releases are non-breaking releases by semver definition, so there is (nearly) no risk of breaking.

> do you want to play safe at the cost of having to make a new release for every Julia version

As I said above, I can install LazySets v1.34.0 despite the fact [`Project.toml`](https://github.com/JuliaReach/LazySets.jl/blob/2c637151709be5968415449769da3d42bb86be25/Project.toml#L42) doesn't mention Julia 1.4

---

You are right. These version specifiers are so confusing because they differ for version 0.x and y.x where y != 0 (see [here](https://julialang.github.io/Pkg.jl/v1/compatibility/#Caret-specifiers-1)). So we would need to use a `~` to disallow newer minor versions.
I will keep things as they are for now because this was just a meta comment but we will consider it. Thanks for the pointer!